### PR TITLE
Ethan's Edits

### PR DESCRIPTION
**Pull request by ethanmpeterson**

### CODE REVIEW

File: reports/final/report.tex [hunk 1]
```diff
@@ -74,14 +74,16 @@ \section*{Executive Summary}
...
-signals and current from a hall-effect sensor. The PCB also has a two-channel 40MSPS
-ADC for observing high speed signals. Both ADCs are connected to an FPGA, which is in
-turn connected to USB 2.0 PHY. The FPGA collects sample data from both ADCs and frames
-it into packets, which are sent over the USB PHY and transmitted to the computer to be
-decoded and displayed. This allows TeachEE to leverage the student's machine to create
-a streamlined user experience. 
+TeachEE's hardware consists of a four-layer custom designed Printed Circuit
+Board (PCB) powered by a USB connection to the computer. The PCB has a
+two-channel 1MSPS ADC for reading low-speed signals and current from a
+hall-effect sensor. The PCB also has a two-channel 40MSPS Analog to Digital
```
Comment (EYangster) [inline reports/final/report.tex]: 40MSPS should be separated with space

File: reports/final/report.tex [hunk 2]
```diff
@@ -1528,9 +1532,11 @@ \subsubsection{Features and Quality}
 In the online electrical engineering labs during the pandemic, an electronics
 professor noticed issues with the kits provided to students. There was a general
 lack of quality and brittleness which caused students to require replacement
-kits. Additionally, the implementation of certain features were poor; the
-display was unable to simultaneously show the frequency and time domain, and it
-was poorly indicated which was currently being displayed. 
+kits. Additionally, the scopes and signal generators had few protections against
```
Comment (EYangster) [inline reports/final/report.tex]: Best to not use "scopes" and just "oscilloscopes"

File: reports/final/report.tex [hunk 3]
```diff
@@ -1528,9 +1532,11 @@ \subsubsection{Features and Quality}
...
+extraneous inputs resulting in students accidentally breaking their tools when
+incorrectly connected. At a functional level, the scopes were missing critical
```
Comment (EYangster) [inline reports/final/report.tex]: scopes

File: reports/final/report.tex [hunk 4]
```diff
@@ -1528,9 +1532,11 @@ \subsubsection{Features and Quality}
 In the online electrical engineering labs during the pandemic, an electronics
 professor noticed issues with the kits provided to students. There was a general
 lack of quality and brittleness which caused students to require replacement
-kits. Additionally, the implementation of certain features were poor; the
-display was unable to simultaneously show the frequency and time domain, and it
-was poorly indicated which was currently being displayed. 
+kits. Additionally, the scopes and signal generators had few protections against
+extraneous inputs resulting in students accidentally breaking their tools when
```
Comment (EYangster) [inline reports/final/report.tex]: Comma after "inputs"

File: reports/final/report.tex [hunk 5]
```diff
@@ -1528,9 +1532,11 @@ \subsubsection{Features and Quality}
...
+incorrectly connected. At a functional level, the scopes were missing critical
+features. They were single channel and had no ability to measure current or
```
Comment (EYangster) [inline reports/final/report.tex]:
```suggestion
features: they were single channel and had no ability to measure current or
```

**Review by EYangster — CHANGES_REQUESTED** [review]

File: reports/final/report.tex (hunk 1 above)
Comment (EYangster) [reply reports/final/report.tex]: Same with 1MSPS

File: reports/final/report.tex [hunk 6]
```diff
@@ -315,14 +318,14 @@ \section{Implementation} \label{sec:impl}
...
-the industry standard tool for PCB design. Our design takes advantage of
-Altium's hierarchical schematics. This allows the schematic to be broken up into
-multiple pages with interconnects between the different sub-circuits.
-TeachEE's top level schematic is an interconnect between all the sub-circuits of
-the PCB, and is shown in Figure
-\ref{fig:hw-top-level-sheet}. All eight pages of the
-schematic are provided in Appendix \ref{appendix:schematic}.
+PCB. The PCB is designed using the Electronic Design Automation (EDA) software
+Altium Designer, the industry standard tool for PCB design. Our design takes
+advantage of Altium's hierarchical schematics. This allows the schematic to be
```
Comment (EYangster) [inline reports/final/report.tex]:
```suggestion
advantage of Altium's hierarchical schematics, allowing the schematic to be
```

File: reports/final/report.tex [hunk 7]
```diff
@@ -433,12 +436,12 @@ \subsubsection{FPGA Module}
...
 physical PCBs had already been manufactured and delivered. To use the
 parallel-bus interface of the FT232HQ to send data, the FPGA must lock to a
-\SI{60}{\mega\hertz} output clock from the FT232HQ. During layout, the FPGA
-pin wired to this clock was swapped to a normal GPIO pin rather than an MRCC
-dedicated clock pin. As a result, the FPGA place and route was failing during
-compilation, as the FPGA could not route this clock from a standard GPIO pin.
-This issue was resolved by waiving the warnings and optimizing the SystemVerilog
-code to still pass timing without a proper clock route for the
+\SI{60}{\mega\hertz} output clock from the FT232HQ. During layout, the FPGA pin
+wired to this clock was swapped to a normal GPIO pin rather than an Multi Region
```
Comment (EYangster) [inline reports/final/report.tex]:
```suggestion
wired to this clock was swapped to a normal GPIO pin rather than a Multi Region
```

**Review by EYangster — CHANGES_REQUESTED** [review]

File: reports/final/report.tex (hunk 2 above)
Comment (ethanmpeterson) [reply reports/final/report.tex]: I agree my bad there

File: reports/final/report.tex (hunk 4 above)
Comment (ethanmpeterson) [reply reports/final/report.tex]: Fixed

File: reports/final/report.tex (hunk 3 above)
Comment (ethanmpeterson) [reply reports/final/report.tex]: fixed

File: reports/final/report.tex (hunk 1 above)
Comment (ethanmpeterson) [reply reports/final/report.tex]: Fixed

**Comment by jagprog5** [comment]
Looks good to me.

**Review by EYangster — APPROVED** [review]

**Pull request merged**